README.md

```diff
@@ -62,7 +62,11 @@ The following table lists the configurable parameters for this chart and their d
 
 | Parameter                                       | Description                                                         | Default                                      |
 | ------------------------------------------------|---------------------------------------------------------------------|----------------------------------------------|
+| `global.imageRegistry`                          | Used for Bitnami PostgreSQL and Redis                               | `""`                                          |
+| `global.imagePullSecrets`                       | Used for Bitnami PostgreSQL and Redis                               | `[]`                                          |
+| `global.storageClass`                           | Used for Bitnami PostgreSQL and Redis                               | `""`                                          |
```

Comment:
These three values are used in NetBox chart as well.
The storage class is indeed not used but should/will be when migrating to `common.storage.class` helper.

Reply:
When I search for `global.` at the repo level of `netbox-chart` I got 0 matches. So I am not sure where you find the references. They should not be removed because they are used in the sub-charts. The description should accurately reflect their usage.

Reply:
* https://github.com/search?q=repo%3Anetbox-community%2Fnetbox-chart%20.global&type=code
* https://github.com/search?q=repo%3Anetbox-community%2Fnetbox-chart+common.images.renderPullSecrets&type=code

😊